command/vsm_stats.go

```diff
@@ -409,3 +292,135 @@ func GetVolumeStats(address string, obj interface{}) (error, int) {
 	rc := json.NewDecoder(resp.Body).Decode(obj)
 	return rc, 0
 }
+
+// StatsOutput Displays the Output in json or default format
+func StatsOutput(c *VsmStatsCommand, annotations *Annotations, args []string, statusArray []string, stats1 VolumeStats, stats2 VolumeStats) error {
+
+	var (
+		err error
+
+		ReadLatency  int64
+		WriteLatency int64
+
+		AvgReadBlockCountPS  int64
+		AvgWriteBlockCountPS int64
+	)
+
+	ReadIOPSi, _ := strconv.ParseInt(stats1.ReadIOPS, 10, 64)
```

Comment:
I really do not believe in short variable names as prescribed by golang community.
i.e. https://github.com/golang/go/wiki/CodeReviewComments#variable-names
We can discuss over slack the best practices that we need to follow for coding in any language.

In this comment, I was thinking about camel casing.